<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PSHelperMainWindow</class>
 <widget class="QMainWindow" name="PSHelperMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="PSDataTableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="DataInfoLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_WriteToCSVFile"/>
    <addaction name="separator"/>
    <addaction name="QuitAction"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="PreferencesAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="AboutAction"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_ShowExtendedInfo"/>
    <addaction name="action_ShowColorsForStatus"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="QueueDockWidget">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>171</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Process Queue</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="margin">
      <number>5</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>478</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="QueueListWidget"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="ClearQueueButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="RunQueueButton">
         <property name="text">
          <string>Run Queue</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="QuitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="PreferencesAction">
   <property name="text">
    <string>Preferences ...</string>
   </property>
   <property name="toolTip">
    <string>Edit the general program preferences</string>
   </property>
  </action>
  <action name="AboutAction">
   <property name="text">
    <string>About PhotoScan Helper</string>
   </property>
   <property name="toolTip">
    <string>Information about PhotoScan Helper</string>
   </property>
  </action>
  <action name="action_ShowExtendedInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Extended Info</string>
   </property>
   <property name="toolTip">
    <string>Display extra columns with more information about each project.</string>
   </property>
  </action>
  <action name="action_ShowColorsForStatus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Colors for Status</string>
   </property>
   <property name="toolTip">
    <string>Color the extended info to indicate the quality/status of each column.</string>
   </property>
  </action>
  <action name="action_WriteToCSVFile">
   <property name="text">
    <string>Write to CSV file ...</string>
   </property>
   <property name="toolTip">
    <string>Save all project data to a CSV file for viewing in a spreadsheet program.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
